### PR TITLE
Feature: Added a play named Random Avatar

### PR DESCRIPTION
**Pull request by hamzathul**
hacktoberfest2024

# Description

This pull request implements a play named Random Avatar generator. The play is a React-based interactive web application that generates random avatars when the "Regenerate" button is clicked.

### Linked Issue:
This pull request addresses and closes Issue #1537  - "[Add a Play]: Random Avatar"

# Conformations:

- The implementation meets the requirements outlined in the issue.
- All features are functioning as expected.
- Code is written in a clear and maintainable way.
- Proper naming conventions are followed.
- Code is properly formatted and adheres to the style guide.
- The README is updated with relevant information about the new features.
- The code contains descriptive commit messages.
- The application works correctly on different screen sizes and devices.

# Screenshots or example output

Screenshot from 2024-10-09 16-12-48

### CODE REVIEW

File: src/plays/random-avatar/styles.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  justify-content: center;
+}
+
+.super-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main {
```
Comment (priyankarpal) [inline src/plays/random-avatar/styles.css:21]: please change the class name

File: src/plays/random-avatar/styles.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  background: #222;
+  position: absolute;
+  width: 60%;
+  height: 100%;
+  left: 20%;
+  top: 0;
+  z-index: -1;
+}
+
+.circle {
```
Comment (priyankarpal) [inline src/plays/random-avatar/styles.css:64]: same here change the name

File: src/plays/random-avatar/RandomAvatar.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
+          {/* Your Code Starts Here */}
+          <div className="super-body super-selection">
+            <div className="super-container">
+              <main className="main">
```
Comment (priyankarpal) [inline src/plays/random-avatar/RandomAvatar.js:43]: change the class name to something that's different/related to your project

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by hamzathul** [comment]
class names are changed


@priyankarpal

**Review by priyankarpal — APPROVED** [review]

**Pull request merged**